SPECS/asc/asc.spec

```diff
@@ -0,0 +1,24 @@
+Summary:        Package for Mariner to meet Azure Security Baseline 
+Name:           asc
+Version:        1.0
+Release:        1%{?dist}
+License:        MIT
+Requires:       filesystem-asc
```

Comment:
Please lint the spec and also follow the instructions from the PR checks.

Reply:
change as suggested